enterprise/backend/test/metabase_enterprise/advanced_permissions/models/group_manager_test.clj

```diff
@@ -0,0 +1,94 @@
+(ns metabase-enterprise.advanced-permissions.models.group-manager-test
```

Comment:
nit: can this be `metabase-enterprise.advanced-permissions.models.permissions.group-manager-test` so that it corresponds to the non-test namespace?